### ISSUE TITLE
Add composable to access roles or permissions

### ISSUE THREAD

**Issue by carpad88** [dystcz/nuxt-permissions#2]
It would be great to have a composable that lets us access the roles or permissions to use it in our logic, for example, to check if a user can execute one function. 

The directives are great for hiding or showing UI parts, but we are limited if we need to execute some specific condition.

For example, let's assume users can see the list of permissions, but if they don't have the permission `update_permissions,` they shouldn't switch the checkboxes, only see them. Something like the following

```vue
<input
  v-model="rolesAndPermissions[action]"
  type="checkbox"
  class="form-checkbox"
  :disabled="!user.permissions.includes('update_roles')"
/>
```

<img width="1180" alt="Screenshot 2023-05-01 at 14 19 24" src="https://user-images.githubusercontent.com/13703238/235523930-a3b755dc-cded-4185-ab60-8f19585e0611.png">

<img width="1178" alt="Screenshot 2023-05-01 at 14 20 48" src="https://user-images.githubusercontent.com/13703238/235524153-829e92d5-e499-4c62-b8ec-20938bc76249.png">

**Comment by crossatko** [comment]
Hi, this should be totally possible with something like:
```vue
<script setup>
  const userPermissions = useCookie('permissions')
  console.log(userPermissions.value.includes('update_roles')) // returns true or false
</script>
```

**Comment by carpad88** [comment]
Actually I'm using this module with [nuxt-sanctum-auth](https://github.com/dystcz/nuxt-sanctum-auth) and I'm getting the permission through the `user` like this:

```ts
const { user } = useAuth()
console.log(user.permissions.include('update_roles')
```

My suggestions is because I saw this module behind the scenes has this methods `hasPermissions` and `hasRoles`, the composable could be to acces them easily, something like this:

```ts
const { hasPermission, hasRole } = usePermissions()
console.log(hasPermission('update_roles')
console.log(hasRole('admin')
```

**Comment by crossatko** [comment]
true, that would be great.
I will look into it in few days, or feel free to submit a PR.
Thanks for the suggestion

**Comment by carpad88** [comment]
After some thinking, I decided to only expose the functions directly in the plugin; it wasn't needed to create a composable.

I already sent the PR, so you can approve it if this approach works.

**Comment by crossatko** [comment]
released [v0.1.4](https://github.com/dystcz/nuxt-permissions/releases/tag/v0.1.4)

I will probable create the composable with more functionality in the future, but for now it works well.
Thanks for your contribution.
